File: php/grab.php
<?php
define("host", "localhost"); //set host
define("database", "kippo"); //database
define("user", "kippo"); //user that uses database
define("password", "Kippo-DB-pass"); //password for that user

//-------------------------GENERATOR CODE------------------------
$mysqli = new mysqli(host, user, password, database);

if (mysqli_connect_errno()) {
    printf("Connect failed: %s\n", mysqli_connect_error());
    exit();
}

if($result = $mysqli->query("select username, password from auth;")) {
    while($row = mysqli_fetch_assoc($result)){
     echo $row['username'].':0:'.$row['password'].PHP_EOL;
    }
    $result->close();
}
$mysqli->close();
//----------------------------END OF CODE------------------------

File: python/grab.py
#!/usr/bin/python
import MySQLdb

db = MySQLdb.connect(host="localhost",    # your host, usually localhost
                     user="kippo",         # your username
                     passwd="Kippo-DB-pass",  # your password
                     db="kippo")        # name of the data base

cur = db.cursor()

cur.execute("SELECT username, password FROM auth")

for row in cur.fetchall():
    print "{}:0:{}".format(row[0], row[1])

db.close()
